core/CMakeLists.txt

```diff
@@ -390,7 +390,9 @@ if (UNIX)
       set(OS_SRCS ${OS_SRCS} unix/loader_android.c)
     else ()
       set(OS_SRCS ${OS_SRCS} unix/loader_linux.c)
-      set(OS_SRCS ${OS_SRCS} unix/coredump.c)
+      if (X64)
+        set(OS_SRCS ${OS_SRCS} unix/coredump.c)
```

Comment:
Put os_dump_core_live() in os_exports.h in X64 as well.

Reply:
I put it under

#if defined(AARCH64) || (defined(X64) && defined(X86))
bool
os_dump_core_live(dcontext_t *dcontext, priv_mcontext_t *mc);
#endif